circle.yml

```diff
@@ -52,6 +52,16 @@ jobs:
           at: /go/src/github.com/gruntwork-io/cloud-nuke
       - run: go run main.go aws --older-than 1h --force
 
+  nuke_sandbox:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /go/src/github.com/gruntwork-io/cloud-nuke
+      - run: export AWS_ACCESS_KEY_ID=SANDBOX_AWS_ACCESS_KEY_ID
+      - run: export AWS_SECRET_ACCESS_KEY=SANDBOX_AWS_SECRET_ACCESS_KEY
+      - run: go run main.go aws --older-than 1h --force
```

Comment:
Sandbox should only be older than 24h, right?

Reply:
Ah thanks. Good catch